Coco_Session2.ipynb

```diff
@@ -1158,7 +1158,7 @@
     "\n",
     "---\n",
     "\n",
-    "¿Qué sucede si intentamos acceder al elemento 10 de la lista `[-3.0, -2.0, -1.0]`?\n",
+    "¿Qué sucede si intentamos acceder al elemento 3 de la lista `[-3.0, -2.0, -1.0]`?\n",
```

Comment:
@andrea This is not an error. the correct value is 10. The idea is to see what the students answer when you try to access element 10 of an array of 3

Reply:
The idea is to show them read an error